usr/lib/ep11_stdll/ep11_specific.c

```diff
@@ -902,9 +943,52 @@ static CK_RV ep11tok_pkey_skey2pkey(STDLL_TokData_t *tokdata, SESSION *session,
         goto done;
     }
 
+    if (wk->token_size > sizeof(wrapped_key)) {
+        TRACE_ERROR("Wrapped key size too big\n");
```

Comment:
"token size too big"? or "wrapped key size too small"? Or simply, as the return code says: "Buffer too small"?